src/api/wallet/composeProvider.ts

```diff
@@ -9,6 +9,8 @@ import providerFromEngine from 'eth-json-rpc-middleware/providerFromEngine'
 import { TransactionConfig } from 'web3-core'
 import { numberToHex } from 'web3-utils'
 import { isWalletConnectProvider, Provider } from './providerUtils'
+import { logDebug, logInfo } from 'utils'
```

Comment:
```suggestion
import { logDebug } from 'utils'
```
broke Travis, in half